fetools-app/src/pages/FontVisualizer.jsx

```diff
@@ -154,20 +153,20 @@ const FontVisualizer = () => {
         />
       </div>
 
-      <div className="flex items-center gap-475 self-stretch">
+      <div className="flex items-center mb-4">
         <h2 className="text-2xl font-bold mb-4">Code</h2>
       </div>
 
-      <div className="flex  gap-12">
+      <div className="flex flex-col lg:flex-row gap-12 w-full">
+        <HtmlCodeGenerator generateHtmlCode={generateHtmlCode} />
+
         <CodeGenerator
           generateCssCode={generateCssCode}
           generateTailwindCode={generateTailwindCode}
           handleCopyIcon={handleCopyClick}
           codeType={codeType}
           setCodeType={setCodeType}
         />
-
-        <HtmlCodeGenerator generateHtmlCode={generateHtmlCode} />
       </div>
     </main>
   );
```

Comment:
```suggestion
  );
  
```
Bump to induce deploy preview.